### PR TITLE
update readme with machine explorer

### PR DESCRIPTION
**Pull request by tylersmalley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,20 @@ The Tailscale extension for VS Code brings no-hassle, secure networking alongsid
 
 ## Features
 
+### Explore remote machines
+
+View and interact with machines on your tailnet within VS Code. Powered by [Tailscale SSH](https://tailscale.com/tailscale-ssh/), you can remotely manage files, open terminal sessions, or attach remote VS Code sessions.
+
+![VS Code with the Tailscale Machine Explorer expanded, showing a tailnet with a variety of services](resources/readme/machine-explorer.png)
```
Comment (tylersmalley) [inline README.md:23]: The VS Code marketplace requires full URL's for images.

```suggestion
![VS Code with the Tailscale Machine Explorer expanded, showing a tailnet with a variety of services](https://raw.githubusercontent.com/tailscale-dev/vscode-tailscale/main/resources/readme/machine-explorer.png)
```

File: README.md [hunk 2]
```diff
@@ -16,6 +16,20 @@ The Tailscale extension for VS Code brings no-hassle, secure networking alongsid
...
+
+#### Connect to a machine in your tailnet
+
+You can start a new terminal session or attach VS Code to a machine for remote development in one click. Hover over the machine you'd like to connect to, and click the <img alt="VS Code Terminal Icon" style="display: inline-block; max-height: 1.25em; position: relative; vertical-align: text-top;" src="resources/readme/terminal.svg"> icon to start a terminal session, or the <img alt="VS Code Remote Explorer Icon" style="display: inline-block; max-height: 1.25em; position: relative; vertical-align: text-top;" src="resources/readme/remote-explorer.svg"> icon to attach the VS Code window to the machine.
```
Comment (tylersmalley) [inline README.md:27]:
```suggestion
You can start a new terminal session or attach VS Code to a machine for remote development in one click. Hover over the machine you'd like to connect to, and click the <img alt="VS Code Terminal Icon" style="display: inline-block; max-height: 1.25em; position: relative; vertical-align: text-top;" src="https://raw.githubusercontent.com/tailscale-dev/vscode-tailscale/main/resources/readme/terminal.svg"> icon to start a terminal session, or the <img alt="VS Code Remote Explorer Icon" style="display: inline-block; max-height: 1.25em; position: relative; vertical-align: text-top;" src="https://raw.githubusercontent.com/tailscale-dev/vscode-tailscale/main/resources/readme/remote-explorer.svg"> icon to attach the VS Code window to the machine.
```

Comment (tylersmalley) [reply README.md:27]: Also updated to a PNG

**Review by tylersmalley — COMMENTED** [review]
LGTM

**Review by samlinville — APPROVED** [review]

**Pull request merged**